package chapter12_error_handling_with_exceptions.task13;

public class Exception2 extends Exception {
}
